src/services/ReportService.ts

```diff
@@ -47,6 +47,9 @@ type DownloadReportPhotoResponsePayload = {
 export type Reports = ListReport[];
 export type ReportFiles = ReportFile[];
 export type ReportPhotos = ReportPhoto[];
+
+export type GetSeedBankV1 = components['schemas']['GetSeedBankV1'];
```

Comment:
thoughts on moving this into src/types/Report?
Often the schema component name may be BE specific which we can rename.. SeedBankV1? ReportSeedBank? SeedBankReport?